scripts/php/generate-html-docs.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
```

Comment:
Is this scripts used anywhere?

Reply:
Right now no, I copied the script from the C# folder. Is it a legacy script that needs to be removed?